charts/chronograf/README.md

```diff
@@ -84,35 +76,39 @@ The following table lists the configurable parameters of the chronograf chart an
 | `tolerations`                | Toleration labels for pod assignment                                                                      | []                                          |
 | `affinity`                   | Affinity settings for pod assignment                                                                      | {}                                          |
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+To configure the chart, do either of the following:
 
-```bash
-helm upgrade --install my-release \
-  --set ingress.enabled=true,ingress.hostname=chronograf.foobar.com \
-    influxdata/chronograf
-```
+- Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example, use the following command:
 
-The above command enables persistence and changes the size of the requested data volume to 200GB.
+  ```bash
+  helm upgrade --install my-release \
+    --set ingress.enabled=true,ingress.hostname=chronograf.foobar.com \
+      influxdata/chronograf
+  ```
 
-Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+  This command enables persistence and changes the size of the requested data volume to 200GB.
```

Comment:
Would the  `persistence.size=200GB` parameter need to be included?

Reply:
```
persistence:
  enabled: true
  size: 200GB
```

or

`--set persistence.enabled=true --set persistence.size=200GB`

Reply:
Thanks @rawkode, it almost reads as if line 89 meant to refer to another code block

Reply:
does it matter which order?

 ```bash
  helm upgrade --install my-release \
    --set persistence.enabled=true --set persistence.size=200GB
    --set ingress.enabled=true,ingress.hostname=chronograf.foobar.com \
      influxdata/chronograf